include/config/config_movement.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+/*********************
+ * MOVEMENT SETTINGS *
+ *********************/
+
+// Fixes Mario's turn ground radius by making it dependent on the analog stick magnitude.
+//#define GROUND_TURN_FIX
+
+// Fixes Mario's turn ground radius by allowing Mario to turn around at any speed.
+// Basically a simpler version of GROUND_TURN_FIX but smoother & closer to vanilla.
+#define GROUND_TURNING_AROUND_FIX
```

Comment:
```suggestion
// Fixes Mario's turn ground radius by making it dependent on the analog stick magnitude.
//#define INSTANT_GROUND_TURN

// Fixes Mario's turn ground radius by allowing Mario to turn around at any speed.
// Basically a simpler version of INSTANT_GROUND_TURN but smoother & closer to vanilla.
//#define INSTANT_GROUND_TURN_LITE
```
- This is not a fix. There is no problem being fixed here. It changes the physics of the game to something different. So it should not be called a "fix".
- Please don't turn these on by default